doc(release): prepare release notes for OSS 23.04 (#2962)
centreon/centreon-documentation: versioned_docs/version-23.04/releases/centreon-os.mdx

## Centreon Web

### 23.04.14

Release date: `January 12, 2024`

<details>
  <summary>Bug fixes</summary>

> - [API] Fixed a visibility issue on configuration/users endpoint.

Reviewer: ```suggestion
- [API] Fixed a visibility issue with the configuration/users endpoint.
```

> - [Authentication] Fixed hardcoded email value that block SSO authentication with OpenId Connect when parameter is not email.

Reviewer: ```suggestion
- [Authentication] Fixed hardcoded email value that blocked SSO authentication with OpenId Connect when parameter was not an email.
```

- [Configuration] Fixed error message displayed in configuration generation after upgrading to 23.04 version.
- [Event Logs] Fixed period in CSV export.

> - [Installation] Fixed issue of poller ID appearing twice in Centreon gorgone configuration.

Reviewer: ```suggestion
- [Installation] Fixed issue of poller ID appearing twice in Centreon Gorgone configuration.
```

> - [Installation] Fixed installation when database user already exists.

Reviewer: ```suggestion
- [Installation] Fixed installation issue occurring when database user already existed.
```

> - [My Account] Fixed account properties edition for local user only.

Reviewer: ```suggestion
- [My Account] Made the name, login and password fields read-only for non-local users so as to prevent issues where users who had edited these properties were locked out of the platform.
```

- [Resource Status] Fixed bad host redirection in the services panel.

> - [Resource Status] Fixed default sorting order resources status: from worse to best.

Reviewer: ```suggestion
- [Resource Status] Fixed default sorting order in Resources status: from worst to best.
```

> - [Web] Fixed an issue of appearing herited contacts/contact groups in notification tab.

Reviewer: ```suggestion
- [Web] Fixed an issue where inherited contacts/contact groups were not displayed in the notifications tab.
```
